main.go

```diff
@@ -4,20 +4,84 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"net/http"
+	"os"
 	"time"
 )
 
-func getNewsContent(url string) bytes.Buffer {
-	fmt.Printf("Performing GET request to %v...\n", url)
+func getWebhookURL() string {
+	if len(os.Getenv("discord")) > 0 {
+		return os.Getenv("discord")
+	}
+	log.Fatal("No discord webhook in environment variable!")
+	return ""
+}
+
+// translate appid to game name
+func getGameName(appid int, responseBytes []byte) string {
+	var respJson appIDTranslator
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [var respJson should be respJSON](https://app.codacy.com/app/matthew.clairmont1/steam-news-alerter/pullRequest?prid=4554288)